pkg/api/avatar/avatar.go

```diff
@@ -105,7 +105,7 @@ func (this *CacheServer) Handler(ctx *macaron.Context) {
 	if avatar.notFound {
```

Comment:
Feels like this is not working as expected or has stopped working sometime ago, but not sure. What is the content of `avatar` here?

Reply:
@marefr What do you think isn't working here? I think at this point, the avatar can either be found or not. If it's found, it should apparently be added to the cache. Maybe we should avoid adding it to the cache though if it was originally retrieved from there?

Reply:
I might be able to come up with a cleaner logic, where we only try to add the avatar to the cache if it wasn't already found there.

Reply:
Cool :+1: